VehicleCode/Input/AnalogSensor.cpp

```diff
@@ -12,18 +12,16 @@
  * Initializes an AnalogSensor object with specified parameters.
  *
  * @param id (int) The unique identifier for the sensor.
- * @param freq (int) The update delay of the sensor in milliseconds.
+ * @param waitTime (int) The update delay of the sensor in milliseconds.
  * @param inPins (int) The input pin id for the analog sensor.
  */
-AnalogSensor::AnalogSensor(int id, int waitTime, int inPin, int bias, int max, int dataLength) {
+AnalogSensor::AnalogSensor(int id, int waitTime, int inPin) {
     sensorID = id;
     this->waitTime = waitTime;
     previousUpdateTime = millis();
     inputPins[0] = inPin;
     sensorValue = 0;
-    this->bias = bias;
-    this->max = max;
-    this->dataLength = dataLength;
+    this->dataLength = 2;
 };
```

Comment:
Are we sure all data will be 2 indices long?

Reply:
Yes but good idea to make sure we limit the max at 9999 just so we don't overload on accident. All analog should read 0-1023 so itll always fit into 2 windows if the first window is 1000s and 100s and then second is 10s and 1s